Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -368,6 +371,38 @@ def download_artifact(url, dir, pr)
     end
   end
 
+  def pr_check_conflicts(name, tap_remote_repo, pr)
+    pull_requests = search_issues("org:#{name}", repo: tap_remote_repo, state: "open", label: "\"long build\"")
+    long_prs = pull_requests.map { |pull_request| pull_request["number"] }
+    long_build_pr_files = {}
+    long_prs.each do |long_pr|
+      GitHub.get_pull_request_changed_files(name, tap_remote_repo, long_pr).each do |pr_file|
+        long_build_pr_files[pr_file["filename"]] = Array(long_build_pr_files[pr_file["filename"]]).push(long_pr)
```

Comment:
I think you can do something like this

```suggestion
    long_build_pr_files = Hash.new { |h, k| h[k] = [] }
    long_prs.each do |long_pr|
      GitHub.get_pull_request_changed_files(name, tap_remote_repo, long_pr).each do |pr_file|
        long_build_pr_files[pr_file["filename"]] << long_pr
```

Reply:
Suggested a more comprehensive refactoring above.